### PR TITLE
Button prepare game

### PR DESCRIPTION
**Pull request by remiriv**
## Proposed changes

---

Ticket https://ogre-d4g.atlassian.net/browse/D4G-177

image

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

## Further comments

---

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Where has this been tested?

---

Docker version: …
Operating System: …
node version: …
postgresql version: …

### CODE REVIEW

File: packages/client/src/modules/administration/Games/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const hasGameStarted = (status: string) => {
+  if (["draft", "ready"].indexOf(status) > -1) {
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/administration/Games/utils.tsx]: @remiriv [nano-detail] ça peut s'écrire avec un `.includes`

**Review by louis-sanna-dev — APPROVED** [review]

File: packages/client/src/modules/administration/Games/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const hasGameStarted = (status: string) => {
+  if (["draft", "ready"].includes(status)) {
```
Comment (Baboo7) [inline packages/client/src/modules/administration/Games/utils.tsx:2]: ❤️

**Comment by louis-sanna-dev** [comment]
@remiriv les boutons me semble un peu bizarre:

- l'icône préparer l'atelier n'est pas centrée verticalement
- c'est n'est pas le même police pour ce deux bouttons

Screenshot 2022-11-15 at 23 24 09

**Comment by remiriv** [comment]
@louis-sanna-eki comment ça l'icône n'est pas centrée? le logo tu veux dire? je vois pas le décalage moi 
effectivement pour la police c'est pas la même c'est peut être parce que y a un lien vs juste un button, je me le note

**Pull request merged**